<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Segmento</string>
  </property>
  <widget class="QLineEdit" name="intervalo_1">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>50</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="canal">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Graficar segmento</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cargarRegistro">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar
registro</string>
   </property>
  </widget>
  <widget class="QPushButton" name="graficar">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Graficar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Intervalo 1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Canal:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Intervalo 2</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="intervalo_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>441</width>
     <height>281</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="plano"/>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="fecha">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>110</width>
     <height>24</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="cedula">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="guardar">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>59</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cedula</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>cargarRegistro</tabstop>
  <tabstop>canal</tabstop>
  <tabstop>intervalo_1</tabstop>
  <tabstop>intervalo_2</tabstop>
  <tabstop>cedula</tabstop>
  <tabstop>fecha</tabstop>
  <tabstop>graficar</tabstop>
  <tabstop>guardar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
